algorithms/loopclosure/loop-closure-handler/include/loop-closure-handler/test/test_loop_closure_handling_test.h

```diff
@@ -0,0 +1,118 @@
+#ifndef LOOP_CLOSURE_HANDLER_TEST_TEST_LOOP_CLOSURE_HANDLING_TEST_H_
```

Comment:
this is basically just copied from what was in the test_loop_closure_hand_test.cpp before so the submapmerging test can inherit from it